django-stubs/core/validators.pyi

```diff
@@ -38,7 +38,7 @@ class URLValidator(RegexValidator):
     tld_re: Any = ...
     host_re: Any = ...
     schemes: Any = ...
-    def __init__(self, schemes: Optional[List[str]] = ..., **kwargs: Any) -> None: ...
+    def __init__(self, schemes: Optional[Iterable[str]] = ..., **kwargs: Any) -> None: ...
```

Comment:
From the implementation, it looks like it should be a `Container`, `Iterable` is not enough. Could you make it a `Collection`, it implements both interfaces?